pages/services/couchbase/1.0.0-6.0.0/uninstall/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: layout.pug
+navigationTitle:  Uninstall
+title: Uninstall
+menuWeight: 55
+excerpt: Uninstalling DC/OS Couchbase Services
+featureMaturity:
+enterprise: false
+model: /services/couchbase/data.yml
+render: mustache
+---
+
+## DC/OS 1.10
+
+If you are using DC/OS 1.10 or later:
+
+Uninstall the service from the DC/OS CLI with the command `dcos package uninstall <package_name>`. For example, to uninstall the {{ model.techName }} instance named `couchbase-dev`, run:
+
+  ```bash
+  dcos package uninstall couchbase-dev
+  ```
+  
+### Uninstall procedure
+
+Uninstalling the service consists of the following steps. The scheduler is relaunched in Marathon with the environment variable SDK_UNINSTALL set to “true”. This puts the Scheduler in an uninstall mode.
```

Comment:
Please capitalize "Scheduler".
Rewrite last sentence: "This puts the Scheduler in uninstall mode."

Reply:
Done.